src/_P076_HLW8012.ino

```diff
@@ -331,7 +338,7 @@ boolean Plugin_076(uint8_t function, struct EventStruct *event, String& string)
       hlwMultipliers[2] = getFormItemFloat(F("powmult"));
 
       if ((hlwMultipliers[0] > 1.0) && (hlwMultipliers[1] > 1.0) && (hlwMultipliers[2] > 1.0)) {
-        SaveCustomTaskSettings(event->TaskIndex, reinterpret_cast<const uint8_t *>(&hlwMultipliers),
+        SaveCustomTaskSettings(event->TaskIndex, reinterpret_cast < const uint8_t * > (&hlwMultipliers),
```

Comment:
The spaces around the angle brackets aren't in our Uncrustify config, did you add them yourself? As Uncrustify will remove them later...

Reply:
No, I did not add them.
I run uncrustify with this command:
```
┌──(siggi@nitro)-[~/Desktop/ESPEasy/ESPEasy]
└─$ uncrustify -c uncrustify.cfg -f src/_P076_HLW8012.ino -o src/_P076_HLW8012.ino
```

Reply:
When using the suggested VSCode/PlatformIO setup, you can format using the Format Document option, (and make Uncrustify the default formatter on first use), so we're all using the same settings.

What version of Uncrustify is installed on your system? as the current configuration requires v0.78 or newer to properly recognize all used settings.

Reply:
I'm not familiar mit PlatformIO so I run uncrustify in the console.
When I'm using the format document option from the menu it's formatting the code like you say. I'm now using this method.